metabench.cmake

```diff
@@ -274,11 +274,11 @@ file(WRITE ${METABENCH_RB_PATH}
 ################################################################################
 set(MEASURE_RB_PATH ${METABENCH_DIR}/measure.rb)
 file(WRITE ${MEASURE_RB_PATH}
-"require 'benchmark'                                                    \n"
-"command = ARGV.join(' ')                                               \n"
-"time = Benchmark.realtime { `#{command}` }                             \n"
-"puts \"[command line: #{command}]\"                                    \n"
-"puts \"[compilation time: #{time}]\"                                   \n"
+"require 'benchmark'                                                         \n"
+"command = ARGV.map{ |arg| arg.match(/\\s/) ? '\"'+arg+'\"' : arg }.join(' ')\n"
```

Comment:
I think something less obvious is happening. Look how the compiler command is messed up when we quote it: https://ci.appveyor.com/project/ldionne/metabench/build/1.0.185#L54